### ISSUE TITLE
[BUG] [Formatter] Incorrect indentation of custom Django template tags with children values

### ISSUE THREAD

**Issue by janbaykara** [Riverside-Healthcare/djLint#472]
- [x] I'm on the [latest version](https://pypi.org/project/djlint/) of djLint
- [X] I've searched the [issues](https://github.com/Riverside-Healthcare/djLint/issues)
- [X] I've read the [docs](https://djlint.com)

## System Info
 - OS: nikolaik/python-nodejs:python3.9-nodejs18-bullseye docker image running [Debian Bullseye](https://www.debian.org/releases/bullseye/)
 - Python Version 3.9.15
 - djLint Version 1.19.7
 - template language: django template language

## Issue

Custom template tags provided by [slippers](https://mitchel.me/slippers/docs/introduction/) are incorrectly indented.

When I write code like this:

```
    {% fragment as label %}
       {% for category in page.category.all %}<span>{{ category }}</span>{% endfor %}
    {% endfragment %}
```

djlint auto-formats it to this:

```
    {% fragment as label %}
    {% for category in page.category.all %}<span>{{ category }}</span>{% endfor %}
{% endfragment %}
```

## How To Reproduce

1. Install `slippers`
2. Use the fragment code as above
3. Run formatting via djlint
4. See that the indenting is off

**Comment by christopherpickering** [comment]
Any chance you can share your config? Have you added `fragment` as a custom block?

**Comment by christopherpickering** [comment]
thanks!

the `fragment` tag should be in the config:

```yaml
[tool.djlint]
profile = "django"
use_gitignore = true
ignore="H006,H031" # Ignore img width/height requirements
custom_blocks="fragment"
```

![image](https://user-images.githubusercontent.com/17788706/205718274-ea4efe98-f9c0-45df-8e08-95123fe69a07.png)

Does that work for you? Thanks for linking the slippers docs, we can probably include thier tags in the default set.

**Comment by janbaykara** [comment]
I see, I didn't realise one had to specify tags for them to be formatted... I assume this won't be necessary after the reimplementation of DJLint with an AST?

**Comment by christopherpickering** [comment]
The tags included with django/jinja/twig/etc are formated by default but custom tags need to be specified right now. We will see what happens in the next edition... still working on that code.

**Comment by janbaykara** [comment]
That makes sense!
